tests/test_io.py

```diff
@@ -0,0 +1,217 @@
+        if p == "Spectra":
+            print("Making Spectra files")
+            fileList = [
+                "Ambient",
+                "AntSim3",
+                "HotLoad",
+                "LongCableOpen",
+                "LongCableShorted",
+            ]
+            for filename in fileList:
+                # print(filename)
+                name1 = filename + "_01_2020_001_01_01_01_lab.acq"
+                file1 = dlist[i] / name1
+                # print(file1)
+                file1.touch()
+        if p == "Resistance":
+            print("Making Resistance files")
+            fileList = [
+                "Ambient",
+                "AntSim3",
+                "HotLoad",
+                "LongCableOpen",
+                "LongCableShorted",
+            ]
+            for filename in fileList:
+                # print(filename)
+                name1 = filename + "_01_2020_001_01_01_01_lab.csv"
+                file1 = dlist[i] / name1
+                # print(file1)
+                file1.touch()
+        if p == "S11":
+            print("Making S11 files")
+            fileList = ["External", "Match", "Open", "Short"]
+            for k, s in enumerate(s11List):
+                slist.append(dlist[i] / s)
+                slist[k].mkdir()
+                if s[:-2] == "ReceiverReading":
+                    fileList = ["ReceiverReading", "Match", "Open", "Short"]
+                elif s[:-2] == "SwitchingState":
+                    fileList = [
+                        "ExternalOpen",
+                        "ExternalMatch",
+                        "ExternalShort",
+                        "Match",
+                        "Open",
+                        "Short",
+                    ]
+                else:
+                    fileList = ["External", "Match", "Open", "Short"]
+                for filename in fileList:
+                    # print(filename)
+                    name1 = filename + "01.s1p"
+                    name2 = filename + "02.s1p"
+                    file1 = slist[k] / name1
+                    file1.write_text(
+                        "# Hz S RI R 50\n40000000        0.239144887761343       0.934085904901478\n40000000        0.239144887761343       0.934085904901478"
+                    )
+                    # print(file1.read_text())
+                    file2 = slist[k] / name2
+                    file2.write_text(
+                        "# Hz S RI R 50\n40000000        0.239144887761343       0.934085904901478\n40000000        0.239144887761343       0.934085904901478"
+                    )
+
+    return b
+
+
+# function to make observation object
+def new_testObs(testdir):
+    testObs = io.CalibrationObservation(testdir)
+    return testObs
+
+
+### directory testing
+def test_make_good_Obs(create_test_env, caplog):
+    # test that correct layouts pass (make an obs)
+    testDir = create_test_env
+    try:
+        new_testObs(testDir)
+    except Exception:
```

Comment:
Don't need the try: except: in a test here, because if it fails, it will print out the exception anyway (and fail the test as we want it to)